examples/exp_configs/rl/multiagent/multiagent_i210.py

```diff
@@ -54,11 +55,13 @@
     num_vehicles=0,
     lane_change_params=SumoLaneChangeParams(lane_change_mode="strategic"),
     acceleration_controller=(IDMController, {"a": .3, "b": 2.0, "noise": 0.5}),
+    car_following_params=SumoCarFollowingParams(speed_mode="no_collide"),
```

Comment:
```suggestion
    car_following_params=SumoCarFollowingParams(
        speed_mode="no_collide",
        min_gap=0,
    ),
```

Reply:
this could be contributing to the growing waves

Reply:
hm ok — would be best to remove this then and leave this decision to the waves team